From 636989bbbf94934387528500d8b99cf756959c18 Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.com>
Date: Thu, 1 Feb 2024 10:03:09 +0000
Subject: [PATCH 0871/1085] ARM: dts: bcm2712-rpi-5-b: Add act_led_gpio

The Pi 5 DTB was lacking the act_led_gpio parameter, and
act_led_activelow as implemented the old way. Fix both.

See: https://forums.raspberrypi.com/viewtopic.php?t=364719

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 arch/arm/boot/dts/broadcom/bcm2712-rpi-5-b.dts | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

--- a/arch/arm/boot/dts/broadcom/bcm2712-rpi-5-b.dts
+++ b/arch/arm/boot/dts/broadcom/bcm2712-rpi-5-b.dts
@@ -824,8 +824,10 @@ spi10_cs_pins: &spi10_cs_gpio1 {};
 		uart0 = <&uart0>, "status";
 		wifiaddr = <&wifi>, "local-mac-address[";
 
-		act_led_activelow = <&led_act>, "active-low?";
+		act_led_gpio = <&led_act>,"gpios:4",<&led_act>,"gpios:0=",<&gpio>;
+		act_led_activelow = <&led_act>,"gpios:8";
 		act_led_trigger = <&led_act>, "linux,default-trigger";
+		pwr_led_gpio = <&led_pwr>,"gpios:4";
 		pwr_led_activelow = <&led_pwr>, "gpios:8";
 		pwr_led_trigger = <&led_pwr>, "linux,default-trigger";
 		eth_led0 = <&phy1>,"led-modes:0";
